src/Commands/Teams/GetTeamsPrimaryChannel.cs

```diff
@@ -0,0 +1,33 @@
+﻿using PnP.Framework.Graph;
+using PnP.PowerShell.Commands.Attributes;
+using PnP.PowerShell.Commands.Base;
+using PnP.PowerShell.Commands.Base.PipeBinds;
+using PnP.PowerShell.Commands.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace PnP.PowerShell.Commands.Teams
+{
+    [Cmdlet(VerbsCommon.Get, "PnPTeamsPrimaryChannel")]
+    [RequiredMinimalApiPermissions("Group.Read.All")]
```

Comment:
This permission seems not to be correct. See https://docs.microsoft.com/en-us/graph/api/team-get-primarychannel?view=graph-rest-1.0&tabs=http#permissions. I believe it should be Channel.ReadBasic.All.

Reply:
Thanks @KoenZomers : I have updated the RequiredMinimalApiPermissions to Channel.ReadBasic.All.